src/core/agent/services/signifyNotificationService.ts

```diff
@@ -102,6 +102,28 @@ class SignifyNotificationService extends AgentService {
         notifications.notes.shift();
       }
       for (const notif of notifications.notes) {
+        await this.markNotification(notif.i);
+        if (notif.a.r === NotificationRoute.MultiSigIcp) {
+          const multisigNotification = await this.signifyClient
+            .groups()
+            .getRequest(notif.a.d);
+          if (!multisigNotification || !multisigNotification.length) {
+            continue;
```

Comment:
I think we should mark here too right? Seems like a "dead" notification if the request isn't in KERIA